asapdiscovery-ml/asapdiscovery/ml/trainer.py

```diff
@@ -729,13 +736,23 @@ def train(self):
                 target = torch.tensor(
                     [[pose[self.target_prop]]], device=self.device
                 ).float()
-                in_range = torch.tensor(
-                    [[pose[f"{self.target_prop}_range"]]], device=self.device
-                ).float()
-                uncertainty = torch.tensor(
-                    [[pose[f"{self.target_prop}_stderr"]]],
-                    device=self.device,
-                ).float()
+
+                if self.has_range:
+                    in_range = torch.tensor(
+                        [[pose[f"{self.target_prop}_range"]]], device=self.device
+                    ).float()
```

Comment:
This is worth noting @kaminow, under current assumptions, we require there to be a `range` and `stderr` tag bit of data alongside. Also these names are non-configurable. I have added a bypass but we can make more flexible in a seperate PR?